spec/integration/file_content_type_validation_integration_spec.rb

```diff
@@ -364,6 +364,11 @@ class Person
       before { subject.avatar = '' }
       it { is_expected.to be_valid }
     end
+
+    context 'invalid string' do
```

Comment:
The example used is not an invalid string. We can say it's an invalid stringified JSON.

Reply:
I will change the description of these tests.